editor/scene_tree_dock.h

```diff
@@ -231,7 +231,7 @@ class SceneTreeDock : public VBoxContainer {
 	virtual void shortcut_input(const Ref<InputEvent> &p_event) override;
 
 	void _new_scene_from(String p_file);
-	void _set_node_owner_recursive(Node *p_node, Node *p_owner);
+	void _set_node_owner_recursive(Node *p_node, Node *p_owner, HashMap<const Node *, Node *> &inverse_duplimap);
```

Comment:
```suggestion
	void _set_node_owner_recursive(Node *p_node, Node *p_owner, HashMap<const Node *, Node *> &r_inverse_duplimap);
```
Use `r_` for return values

Reply:
But it's not return value.
I think you could make it const 🤔